.github/workflows/pr-test-notifier.yml

```
@@ -0,0 +1,27 @@
+permissions:
+  pull-requests: write
+
+jobs:
+  notify:
+    name: Post Test Instructions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Post comment with test trigger instructions
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)